packages/ui5-application-writer/src/data/defaults.ts

```diff
@@ -62,6 +62,7 @@ export const enum UI5_DEFAULT {
     OPENUI5_CDN = 'https://openui5.hana.ondemand.com',
     TYPES_VERSION_SINCE = '1.76.0',
     TYPES_VERSION_PREVIOUS = '1.71.18',
+    TYPES_VERSION_BEST = '1.102.7',
```

Comment:
Could we have some comments to explain what `BEST` means as its a qualitative term, better than ...?

Reply:
Good point, I am also happy to rename the entry to something that fits better. Any suggestions?

What I mean with `BEST` here is, that this is the latest version that will receive downports. Let's assume we have an issue in UI5 version 1.106, then we will fix it with the next available version e.g. 1.108 but if the issue is important, we will also downport it to 1.102 as long as it is supported (Q1 2023). Afterwards, we switch this variable to the new `BEST` version. This is also aligned with our fix strategy for `@sap/ux-specification`

Reply:
Thx I see you added a comment here: https://github.com/SAP/open-ux-tools/pull/670/files#diff-0f207248944a1732a5b7343b9d1bb29e22bbae5fa2acce7a573a344c04c856d2R123